tasks/deploy.yml

```diff
@@ -184,13 +184,24 @@
     - inventory_hostname == groups['fs'][0]
     - not magento_fresh_install|bool
 
-- name: Generate app/etc/env.php
+- name: Generate app/etc/env.php (with redis)
+  template:
+    src="env.redis.php.j2"
+    dest="{{ nex_app_doc_root }}/app/etc/env.php"
+  become: yes
+  become_user: "{{ nex_app_unix_user }}"
+  when:
+    - lb_redis_backnet_addrs is defined
+    - inventory_hostname == groups['fs'][0]
```

Comment:
No, the "lb_redis_backnet_addrs is defined" means there's a redis load balancer in the inventory, and therefore we should configure magento to connect to the lb_redis server.  This happens on the fs server since that's where the files live.  Make sense?